### PR TITLE
Disable web snippet on DEBUG instances

### PR DESCRIPTION
**Pull request by Twixes**
## Changes

We disabled Python analytics on DEBUG instances, but forgot to do the same with the web snippet, resulting in frontend events causing noise in our data. This fixes that.

### CODE REVIEW

File: frontend/src/scenes/sceneLogic.js [hunk 1]
```diff
@@ -124,7 +124,7 @@ export const sceneLogic = kea({
     },
     listeners: ({ values, actions }) => ({
         setScene: () => {
-            window.posthog && window.posthog.capture('$pageview')
+            window.posthog && window.posthog?.capture('$pageview')
```
Comment (paolodamico) [inline frontend/src/scenes/sceneLogic.js]: I think `window.posthog &&` would be extra now as you have the `?`

File: frontend/src/scenes/setup/Setup.js [hunk 2]
```diff
@@ -23,7 +23,7 @@ export const Setup = hot(_Setup)
 function _Setup() {
     const { user } = useValues(userLogic)
     const { location } = useValues(router)
-    const showSessionRecord = window.posthog && window.posthog.isFeatureEnabled('session-recording-player')
+    const showSessionRecord = window.posthog && window.posthog?.isFeatureEnabled('session-recording-player')
```
Comment (paolodamico) [inline frontend/src/scenes/setup/Setup.js]: Same as above

**Review by paolodamico — APPROVED** [review]

File: frontend/src/scenes/sceneLogic.js (hunk 1 above)
Comment (Twixes) [reply frontend/src/scenes/sceneLogic.js]: Correct, removed that.

**Comment by macobo** [comment]
> We disabled Python analytics on DEBUG instances, but forgot to do the same with the web snippet, resulting in frontend events causing noise in our data. This fixes that.

Note this was intentional on the part of session recording project. Will do a follow-up here. :)

**Pull request merged**